Antidote/CallsManager.m

```diff
@@ -20,6 +20,8 @@
 #import "TabBarViewController.h"
 #import "AbstractCallViewController.h"
 
+#define LOG_IDENTIFIER @"CallsManager"
```

Comment:
It would be better to define identifier as `self` as this way address of CallsManager will be logged. It may be useful to distinguish different instances of manager in logs.